bson/src/main/org/bson/util/CopyOnWriteMap.java

```diff
@@ -166,19 +166,14 @@
      * Create a new {@link CopyOnWriteMap} with the supplied {@link Map} to initialize the values.
      *
      * @param map the initial map to initialize with
-     * @deprecated since 0.0.12 use the versions that explicitly specify View.Type
      */
-    @Deprecated
```

Comment:
No need to deprecate since the class itself isn't public

Reply:
Ack